crytic/slither
# fix: don't generate initial value for function variables

### Notes

Previously, code handling default value generation crashed when trying to generate default values for variables with function types. We're not sure how to deal with default function values yet, so this PR provides a short-term fix for the issue by avoiding any attempt to generate default values for function types.

### Testing
* Run `./evaulate.sh run 100` in `tool-exec` and verify that all projects succeed
* Run `make test`

Try running slither on this test file:
```
contract Test {
   function bar(function(uint256, uint256) view returns (uint256) z) internal {

   }
   function test(uint256 a, uint256 b) internal view returns (uint256) {
    return a;
   }

   function a() internal returns(function(uint256, uint256) view returns (uint256)) {

   }

   function foo(uint8[3] memory a) external {
        function(uint256, uint256) view returns (uint256) op = test;
   }
}
```


### Related Issue
https://github.com/CertiKProject/slither-task/issues/250
